core/blockchain_impl.go

```diff
@@ -354,7 +354,8 @@ func newBlockChainWithOptions(
 			NoBuild:    bc.cacheConfig.SnapshotNoBuild,
 			AsyncBuild: !bc.cacheConfig.SnapshotWait,
 		}
-		bc.snaps, _ = snapshot.New(snapconfig, bc.db, bc.triedb, head.Hash())
+		fmt.Println("loading/generating snapshot...")
```

Comment:
should we have this log as part of the harmony system log ?

Reply:
not all user are looking at the console logs .. since it may take a long time, let's also add a log in the system log.

Reply:
> not all user are looking at the console logs .. since it may take a long time, let's also add a log in the system log. 

True, I will add a log for system log as well

Reply:
> not all user are looking at the console logs .. since it may take a long time, let's also add a log in the system log.

done